client/src/containers/EditUserProfile/styles.css

```diff
@@ -1,65 +1,58 @@
+.edit-user-profile-container {
+    max-width: 600px;
+    margin: 0 auto 75px auto;
+}
 .edit-user-profile-container .avatar {
     text-align: center;
     margin-top: 30px;
 }
-
 .edit-user-profile-container .avatar img {
```

Comment:
You should not change inner styles of **UserAvatar** component in the other component.
Default behavior of UserAvatar is render image. This component take all available place (with `max-widht` property)
If you want to change width use props to set width for image.
Or, what I prefer more, wrap **UserAvatar** into div with class, which sets width.

Reply:
Great point! Thank you! I missed this selector! 👍